.github/workflows/perf.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This is a workflow which runs the performance tests with profiling enabled.
+
+name: Perf Profile
+
+on:
+  # Permit manual runs of the workflow.
+  workflow_dispatch:
+
+concurrency:
+  # Cancel any CI/CD workflow currently in progress for the same PR.
+  # Allow running concurrently with any other commits.
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
per-workflow run uniqueness: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow